### DISCUSSION TITLE
Add better documentation navigation and links

### DISCUSSION THREAD

**Discussion by LowLevelKiwi1** [directus/directus#20620] [Feature Requests]
### Summary

The feature that i would love to have in directs is quick links to the directus docs for different things. like say your trying to create some data using the directus flows but you can't remember how the payload should be laid out, having a quick link to the directus docs for this and other features would help tremendously, this will also reduce the struggle of trying to navigate the directus docs when trying to do something on directus, this would be especially useful in flows where some of the information about them in the docs is really hard to find as they are mixed in with other directus features

### Motivation

By adding these changes you will make it a lot easier for beginners to use directus by making the docs more accessible and you will make the docs less imposing as it will be easier to navigate to where you need to go

### Detailed Design

one way you could add this feature would be to add a button next to the header of different items / features that will lead people to the specified page in the directus docs for that item, and if users find this annoying it could be added as a feature that users could turn on and off, thus it won't impact users that won't use this feature

### Must Have:
- Allow Users to quickly access the directus docs for a particular feature
- must send users to the specific features page and not just to the directus docs

### Should Have:
- an easy to find button for users to use

### Could Have:
- an way for users in the settings to turn the feature off if they so wish

### Won't Have:
- E

### Drawbacks

some drawbacks would be that it will take time for this feature to be developed and it will also require someone at directus to navigate their docs in order to make sure that the links direct people to the right page, this will be a time consuming effort for that employee. however this will reduce the amount of time needed to teach people the directus system as they will be able to find what they need a lot easier

### Alternatives

one alternative that could be done is to make the directus doc a lot easier to navigate and use so that people will be able to find the feature they are looking for

### Adoption Strategy

Existing directus developers may find this change to be helpful, but they may also find that the location of the button to be obnoxious, but if you implement the option to turn off this feature then the existing developers will have the choice on whether they want to have it on or not

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
